bin/update_clinvar_schema.py

```diff
@@ -0,0 +1,108 @@
+import gzip
+import argparse
+import sys
+import urllib.request
+import subprocess
+import shutil
+import os
+
+SCHEMA_LOCATION_ATTRIBUTE = 'SchemaLocation'
+
+def launch():
+	parser = ArgParser(sys.argv)
```

Comment:
Please check that this file's format is compliant with PEP-8 (e.g. 4 spaces instead of tabs). @tomdcsmith knows how to configure PyCharm.